### DISCUSSION TITLE
Too complicated.

### DISCUSSION THREAD

**Discussion by Soraiko** [google/mozc#1390] [General]
You gotta be kidding. Just post built binaries. People need to execute 100 commands and install 100 programs to build your tool. You like to spend hours to install a program ? Then why you let people do this ?

**Comment by hiroyuki-komatsu** [comment]
If you need a binary, there are packages by Linux distributions, or Google Japanese Input for Windows and macOS.

**Comment by vdheuvel** [comment]
Agree with soraiko. I'm here because I want to try a privacy friendly alternative to google japanese input. Privacy-friendly shouldn't have to mean I have to spend 1-2 hours installing every app.
